saleor/graphql/account/mutations/authentication.py

```diff
@@ -195,16 +195,16 @@ class Arguments:
             required=False,
             description=(
                 "CSRF token required to refresh token. This argument is "
-                "required when refreshToken is provided as a cookie."
+                "required when `refreshToken` is provided as a cookie."
             ),
         )
 
     class Meta:
         description = (
             "Refresh JWT token. Mutation tries to take refreshToken from the input."
```

Comment:
There is a missing space:
```suggestion
            "Refresh JWT token. Mutation tries to take refreshToken from the input. "
```